src/model.rs

```diff
@@ -106,7 +116,7 @@ pub(crate) struct Status {
     /// Maps a component name to status
     pub(crate) components: BTreeMap<String, ComponentStatus>,
     /// Components that appear to be installed, not via bootupd
-    pub(crate) adoptable: BTreeMap<String, ContentMetadata>,
+    pub(crate) adoptable: BTreeMap<String, Adoptable>,
```

Comment:
I think this change breaks the "stable format" premise stated in the docstring, but current CI does not catch this because the entry/map in `example-status-v0.json` fixture is empty.

Is that a problem, or is this breaking change still ok? To my understanding this data-structure is not persisted to disk, and the change only affects client compatibility.

Reply:
Yeah, this is a valid concern and you're right this sneaks by our tests today.

In the end, while we have `bootupctl status --json` I think the project is still in a "stabilizing" state - there aren't any known consumers of the JSON yet.  

Hmm, probably we should add a simple "is up to date" API, since that's all that most consumers would care about.